docs/site/soap-calculator-tutorial.md

```diff
@@ -0,0 +1,90 @@
+---
+lang: en
+title: 'SOAP calculator Web Service integration tutorial'
```

Comment:
we can replace this whole page with just this yaml:
```
---
lang: en
title: 'SOAP calculator Web Service integration tutorial'
keywords: LoopBack 4.0, LoopBack 4
layout: readme
source: loopback-next
file: examples/soap-calculator/README.md
sidebar: lb4_sidebar
permalink: /doc/en/lb4/soap-calculator-tutorial.html
summary: Integrating a Calculator SOAP web service with LoopBack 4.
---
```

We have a script in `loopback.io` where we reuse the README files from the specified repos. Note that we need to update https://github.com/strongloop/loopback.io/blob/gh-pages/update-v4-readmes.sh in order for this to work.

Reply:
exactly, that's the reason i placed it in its own file at least for now, because when i was trying to build the site locally i received the error saying it couldn't locate the README.md.